src/classes/PMT_Payment_TDTM.cls

```diff
@@ -44,18 +103,51 @@ public class PMT_Payment_TDTM extends TDTM_Runnable {
     * @param objResult the describe for Opportunity 
```

Comment:
remove the objResult comment since it isn't part of the method signature

Reply:
Got this on the last commit.